solidity/contracts/ECDSAKeepRewards.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^0.5.4;
+
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/SafeERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "@keep-network/keep-core/contracts/Rewards.sol";
+
+contract ECDSAKeepRewards is Rewards {
+    uint256 constant MAX_UINT20 = 1 << 20 - 1;
```

Comment:
where does this get used? I couldn't find it in keep-ecdsa or keep-core?

Reply:
Same here, it looks unused so I'll drop it.